Cargo.toml

```diff
@@ -22,7 +22,7 @@ cfg-if = "1.0.0"
 clap = { version = "4.0.8", features = ["cargo", "derive", "env"] }
 csscolorparser = "0.6.2"
 derive-new = "0.5.9"
-dirs = "4.0.0"
+dirs = "5.0.0"

```

Comment:
Is there a reason for using `dirs` instead of the library that we already are using `xdg`?

Perhaps Windows support? But in that case I think we can remove the xdg library, and use only `dirs`

Reply:
Indeed, `dirs` also has Windows support. We could probably replace our `xdg` usage with `dirs`.
Should I do it in this PR or we'll leave that for a separate one?

Reply:
One weird thing with `dirs` is on macOS, `config_dir()` refers to `$HOME/Library/Application Support`, though you probably want it to be under `$HOME/.config`. 

Same thing with caches and local/share, dirs uses `$HOME/Library/Caches` and `$HOME/Library/Application Support` when most people expect it to be `~/.cache` and ~/.local/share`

I think it makes more sense to keep using `xdg` for linux/mac then "hardcode" the path for windows

Reply:
I'm not familiar with macOS directory structure, but this makes sense. I changed it to use `xdg` on mac/linux